testing/src/main/java/org/oppia/testing/FakeSystemClock.kt

```diff
@@ -11,16 +12,19 @@ import javax.inject.Inject
  * consistent way.
  */
 class FakeSystemClock @Inject constructor() {
```

Comment:
Nit for TODO at top of this file:
```
// TODO(#89): Actually finish this implementation so that it properly works across Robolectric and
//  Espresso, and add tests for it.
```

Reply:
I don't have a strong preference for this, so done.

Reply:
Okay. Thanks.